.strict-typing

```diff
@@ -36,6 +36,7 @@ homeassistant.util.location
 homeassistant.util.logging
 homeassistant.util.process
 homeassistant.util.unit_system
+script.hassfest.*
```

Comment:
Can you please keep `.strict-typing` and `mypy.ini` changes for a follow-up PR (and probably need to adjust the pre-commit and CI accordingly)

```suggestion
```